Classes/Domain/Timestamp.php

```diff
@@ -1,32 +0,0 @@
-<?php
-namespace Neos\Cqrs\Domain;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-/**
- * Timestamp
- */
-class Timestamp
```

Comment:
RIght, I wasn't aware of this. Maybe we can incorporate that into `Now` and make it an [DateTimeImmutable](https://github.com/neos/flow-development-collection/blob/master/TYPO3.Flow/Classes/TYPO3/Flow/Utility/Now.php#L26) while we're at it